content/blog/how-to-open-new-file-in-new-tab-always-in-vscode/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 'How to open files always in a new tab in Visual Studio Code'
+date: "2020-01-18"
+slug: 'how-to-open-files-always-in-a-new-tab-in-visual-studio-code'
+description: Solution to open files always in a new tab in Visual Studio Code
+  - vscode
+  - tricks
+show: true
+author: shivam
+---
+
+
+If you are Visual Code fan, and use it as your primary editor to write the code it always sucks to open a file in new tab, everytime you have to either right click on the tab and choose `keep open` or use the shortcut `cmd + k` `enter`. This all happens because of the preview mode of VS Code.
```

Comment:
```suggestion
Visual Code is am amazing editor and I sue it for my everyday code. One thing bug me everytime is the files are not open permanently until I use the shortcut `cmd + k` `enter` or right click and select `keep open` on the filename. This happens because VS Code opens the files by default in `preview mode`, so whenever the new file name is clicked from explorer, the opened file closes and new file is visible.
But if you are equally frustrated by this behavior here is what you need to do. 
```